ush/python/pygfs/task/aero_emissions.py

```diff
@@ -75,10 +105,261 @@ def execute(workdir: Union[str, os.PathLike], aprun_cmd: str) -> None:
         -------
         None
         """
+        AerosolEmissions.run(workdir)
+
+    @classmethod
+    @logit(logger)
+    def run(cls, workdir: Union[str, os.PathLike], current_date: str = None, forecast_dates: list = None, Config_dict: Dict = {}) -> None:
+        emistype = Config_dict.emistype
+        ratio = Config_dict.ratio
+        climfiles = sort(glob("{}{}".format(Config_dict.climfile_str, "*.nc")))
+        print(Config_dict.data_out['copy'][0][0])
+
+        if emistype.lower() == "qfed":
+            basefile = glob("qfed2.emis_*.nc4")
+        elif emistype.lower() == "gbbepx":
+            basefile = glob("GBBEPx_all01GRID.emissions_v*.nc")
+        elif emistype.lower() == "hfed":
+            basefile = glob("hfed.emis_*.x576_y361.*nc4")
```

Comment:
I think we can provide the regex string to `glob` in the yaml.
This section will then simply be:
```python
basefile = glob(emis_glob_str)
```

Reply:
I guess I'm a little confused by this because the glob would need to occur after the initialization method due to the files getting synced in that step. So I don't know if we could glob it within the yaml file itself